src/client/src/ui/app.js

```diff
@@ -16,20 +16,20 @@ class App extends React.Component {
     render() {
         return (
             <div>
-            <JumboTron extraClasses={[]} url="http://i.imgur.com/g65f3jk.png">
-                <div className="row margin-top-md">
-                    <div className="col-md-6 col-md-offset-2">
-                        <JumboSearch/>
+                <JumboTron extraClasses={[]} url="http://i.imgur.com/g65f3jk.png">
```

Comment:
Yes!!! I was fighting with Atom to get this auto formatted, I wrote a lot of the markup in codepen first